ibexa/commerce/5.0/encore/package.json

```diff
@@ -1,32 +1,33 @@
 {
   "devDependencies": {
-    "@hotwired/stimulus": "^3.0.0",
-    "@symfony/stimulus-bridge": "^3.0.0",
-    "@symfony/webpack-encore": "^3.0.0",
     "@babel/preset-react": "^7.0.0",
-    "core-js": "^3.0.0",
-    "regenerator-runtime": "^0.13.2",
-    "webpack-notifier": "^1.6.0",
-    "react-collapsible": "^2.5.0",
-    "sass": "^1.49.7",
-    "sass-loader": "^13.0.0",
     "@babel/runtime": "^7.13.10",
-    "postcss-loader": "^4.3.0",
-    "raw-loader": "^4.0.1",
-    "style-loader": "^2.0.0",
-    "file-loader": "^6.0.0",
     "@ckeditor/ckeditor5-alignment": "^40.1.0",
     "@ckeditor/ckeditor5-build-inline": "^40.1.0",
+    "@ckeditor/ckeditor5-code-block": "^40.1.0",
     "@ckeditor/ckeditor5-dev-utils": "^39.0.0",
-    "@ckeditor/ckeditor5-widget": "^40.1.0",
     "@ckeditor/ckeditor5-theme-lark": "^40.1.0",
-    "@ckeditor/ckeditor5-code-block": "^40.1.0",
+    "@ckeditor/ckeditor5-widget": "^40.1.0",
+    "@hotwired/stimulus": "^3.0.0",
+    "@ibexa/ts-config": "https://github.com/ibexa/ts-config-ibexa#~v1.1.0",
     "@svgr/webpack": "^8.1.0",
-    "ts-loader": "^9.5.1",
-    "@types/react": "^18.3.11",
+    "@symfony/stimulus-bridge": "^3.0.0",
+    "@symfony/webpack-encore": "^3.0.0",
     "@types/react-dom": "^18.3.1",
+    "@types/react": "^18.3.11",
```

Comment:
newest version of @types/react is `19.1.3` WDYT?

Reply:
Good idea, but it first requires updating react to 19.x itself :)